<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>573</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>566</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="CPU">
     <attribute name="title">
      <string>CPU</string>
     </attribute>
     <widget class="QProgressBar" name="pBarCPU">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>461</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Usage (%)</string>
      </property>
     </widget>
     <widget class="QWidget" name="graphWidget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Memory">
     <attribute name="title">
      <string>Memory</string>
     </attribute>
     <widget class="QProgressBar" name="pBar_PC">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>611</width>
        <height>161</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory Usage (%)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory Usage (GB)</string>
      </property>
     </widget>
     <widget class="QWidget" name="memWidget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>621</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Disk">
     <attribute name="title">
      <string>Disk</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Disk Usage (%)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Disk Usage (GB)</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="pBar_DPC">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>611</width>
        <height>161</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="pBar_DGB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::BottomToTop</enum>
      </property>
      <property name="format">
       <string>%v GB</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Network">
     <attribute name="title">
      <string>Network</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>561</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Packet Sent</string>
      </property>
      <widget class="QLCDNumber" name="lcdNumSent">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="digitCount">
        <number>10</number>
       </property>
       <property name="intValue" stdset="0">
        <number>10</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>561</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Packet Received</string>
      </property>
      <widget class="QLCDNumber" name="lcdNumRecv">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>10</number>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
